api/v1/cluster_types.go

```diff
@@ -315,6 +315,10 @@ type ClusterSpec struct {
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 
+	// EmptyDirLimit set the limit for the ephemeral volumes
+	// +optional
+	EmptyDirLimit *string `json:"emptyDirLimit,omitempty"`
```

Comment:
I do not like this field name. What about

```suggestion
	EmptyDirLimit *string `json:"ephemeralVolumeSizeLimit,omitempty"`
```

Reply:
I like `ephemeralVolumeSizeLimit` too